holidays/countries/benin.py

```diff
@@ -0,0 +1,164 @@
+        * [2025 Vodoun Festival](https://web.archive.org/web/20250422034718/https://travail.gouv.bj/page/communiques/les-journees-du-jeudi-9-janvier-2024-et-vendredi-10-janvier-2024-sont-feriees-chomees-et-payees-sur-toute-letendue-du-territoire-national)
+    """
+
+    country = "BJ"
+    default_language = "fr_BJ"
+    # %s (observé).
+    observed_label = tr("%s (observé)")
+    # %s (estimé).
+    estimated_label = tr("%s (estimé)")
+    # %s (observé, estimé).
+    observed_estimated_label = tr("%s (observé, estimé)")
+    supported_languages = ("en_US", "fr_BJ")
+    start_year = 1960
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self, show_estimated=islamic_show_estimated)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("Fête du Nouvel An"))
+
+        if self._year >= 1998:
+            # Vodoun Festival.
+            name = tr("Journée Vaudoun")
+            if self._year >= 2025:
+                self._add_observed(
+                    self._add_holiday_2nd_fri_of_jan(name),
+                    rule=FRI_TO_PREV_THU,
+                )
+            else:
+                self._add_holiday_jan_10(name)
+
+        if 1979 <= self._year <= 1989:
+            # Martyrs' Day.
+            self._add_holiday_jan_16(tr("Journée des Martyrs"))
+
+        if 1979 <= self._year <= 1989:
+            # Youth Day.
+            self._add_holiday_apr_1(tr("Journée de la Jeunesse"))
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Lundi de Pâques"))
+
+        # Labour Day.
+        self._add_labor_day(tr("Fête du Travail"))
+
+        if self._year >= 1990:
+            # Ascension Day.
+            self._add_ascension_thursday(tr("Jour de l'Ascension"))
+
+        # Whit Monday.
+        self._add_whit_monday(tr("Lundi de Pentecôte"))
+
+        if self._year >= 1990:
+            # Assumption Day.
+            self._add_assumption_of_mary_day(tr("Jour de l'Assomption"))
+
+        if 1979 <= self._year <= 1989:
+            # Day of the Popular Armed Forces.
+            self._add_holiday_oct_26(tr("Fête des Forces Armées Populaires du Bénin"))
+
+        if self._year >= 1990:
+            # All Saints' Day.
+            self._add_all_saints_day(tr("La Toussaint"))
+
+        # National Day.
+        name = tr("Fête Nationale")
+        if self._year <= 1990:
+            self._add_holiday_nov_30(name)
+        else:
+            self._add_holiday_aug_1(name)
```

Comment:
```suggestion
        if self._year >= 1991:
            self._add_holiday_aug_1(name)
        else:
            self._add_holiday_nov_30(name)
```
It's better to begin from _latest_ version.